### ISSUE TITLE
Agricraft quests no completing

### ISSUE THREAD

**Issue by Theta1996** [Infinity-Evolved-Reloaded-Modpacks/Infinity-Evolved-Reloaded#273]
**General Info (please complete the following information):**
 - Modpack version: 4.2.0
 - Client or server: client 
 - Packmode: expert
 - Own modifications to the modpack: Just Enough Drags mod added

**Describe the bug**
Working through crop breeding to get necessary seeds for a watering can, I noticed that although I have completed the prerequisites and have obtained the seeds ( say for example carrots and wheat crossed to make sugarcane seeds), the quest hasn't been marked as complete. Having manually marked the quest as complete I have noticed that it is hit and miss as to whether other plant quests auto-complete or not while cross-breeding

**To Reproduce**
E.g.:
1. Obtain wheat and carrot seeds and plant on Agricraft crop sticks
2. Crossbreed to produce sugar cane.
3. Quest doesn't automatically mark as complete.

**Screenshots**
[If applicable, add screenshots to help explain your problem.]

**Crash-log**
[If applicable, add a crash-log. Usually a "crash-[date]-[time]-server/client.txt"]

**Comment by saovenn** [comment]
Hey, thanks for the report. Unfortunately, we are encountering issues with FTBQuests and AgriCraft (Seed's) NBT tags, which are preventing automatic completion / detection.

**Comment by Theta1996** [comment]
All the seeds are 1/1/1 but still not being detected. I just did a quick test and setting the task item to ignore nbt data in the quest book seems to fix the issue.

**Comment by saovenn** [comment]
seems so that the sugarcane quest is working for me 🤔 

also the problem is, every agricraft seed has the same ItemStack ( **<agricraft:agri_seed>** which follows with an exact NBT for all of the three growth stages which are completely needed at the same

> .withTag({agri_analyzed: 0 as byte, agri_strength: 1 as byte, agri_gain: 1 as byte, agri_seed: "vanilla:sugarcane_plant", agri_growth: 1 as byte})

so if you breed it and it has **agri_strength** at 2 it will not be detected.

if you change the NBT Tag from **match** to **ignore** it will only take **<agricraft:agri_seed>** which all AgriCraft seeds begin with.

**Comment by Theta1996** [comment]
Yeah, I found that out the hard way. Since there aren't any rewards tied with the quests, it's not a big deal. I'll keep playing with it and see if I can come up with anything. Though I looking at it at the moment, it looks like the only real way of getting it to work is by manually submitting the quest which I'm not totally against, but would be nice to have it working automatically

**Comment by Theta1996** [comment]
After some playing around in creative, it looks like the quests work as they're meant to (within the limitations of crop stats). Turns out I'm just stupid as I didn't realise some of the seeds I was using were from Pam's gardens.

**Comment by saovenn** [comment]
might be changing it to the end product of those seeds and not the seed itself 🤔

**Comment by Theta1996** [comment]
That could work as we wouldn't have to worry about the stats of the seed messing things up. Waiting for the seeds to grow also wouldn't be long, maybe add something about Agricraft's irrigation system to boost growth times without having to go and farm a bunch of bone meal?
